src/olympia/amo/tests/test_commands.py

```diff
@@ -781,12 +781,19 @@ def test_full_run_typical_response(self):
 
     def _test_full_run_typical_response(self):
         expected_below = (
-            'The following locales are below threshold of 40% or completely '
+            'The following locales are below threshold of 70% or completely '
             'absent in one of our projects in Pontoon:\n- '
             + '\n- '.join(
                 (
+                    'Bulgarian [bg]',
```

Comment:
Is this testing the actual completion rate based on the current locales? Could that be mocked so we wouldn't need to change it when local ratios change?

Reply:
It's using mocked data instead of the actual source, which is pontoon API. Because that mocked data hasn't changed, the test had to be updated to match.